test/integration/stats_integration_test.cc

```diff
@@ -383,7 +383,7 @@ TEST_P(ClusterMemoryTestRunner, MemoryLargeClusterSizeWithRealSymbolTable) {
   EXPECT_MEMORY_LE(m_per_cluster, 38000); // Round up to allow platform variations.
 }
 
-TEST_P(ClusterMemoryTestRunner, MemoryLargeHostSizeWithStats) {
+TEST_P(ClusterMemoryTestRunner, DISABLED_MemoryLargeHostSizeWithStats) {
```

Comment:
Could we instead disable the EXPECT_MEMORY_EQ's but keep the EXPECT_MEMORY_LE's?

Reply:
Sure